src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -5,32 +5,48 @@
   <PropertyGroup>
     <SharedFrameworkValidationRefPath>$(RefRootPath)SharedFrameworkValidation</SharedFrameworkValidationRefPath>
     <SharedFrameworkValidationRuntimePath>$(BinDir)runtime/SharedFrameworkValidation</SharedFrameworkValidationRuntimePath>
+    <_PublishRID Condition="'$(OSGroup)'=='Windows_NT'">win-$(ArchGroup)</_PublishRID>
+    <_PublishRID Condition="'$(OSGroup)'=='OSX'">osx-$(ArchGroup)</_PublishRID>
+    <_PublishRID Condition="'$(OSGroup)'=='Linux'">linux-$(ArchGroup)</_PublishRID>
+    <_PublishRID Condition="'$(_PublishRID)'==''">unix-$(ArchGroup)</_PublishRID>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
+   <_MicrosoftNetCoreAppPackageName>Microsoft.NETCore.App</_MicrosoftNetCoreAppPackageName>
```

Comment:
sounds good. Actually I was checking and it looks like all packages that get restored by dotnet into packages dir are lowercase, I'm not sure if this is by design or it's just something with the current CLI that we are using.